packages/java/endpoint/src/main/java/com/vaadin/hilla/EndpointControllerConfiguration.java

```diff
@@ -154,6 +155,7 @@ private static ObjectMapper createDefaultEndpointMapper(
      * @return the endpoint invoker
      */
     @Bean
+    @DependsOn("hillaEndpointObjectMapper")
```

Comment:
This ensures that the bean named ”hillaEndpointObjectMapper” is created before this bean, right? But what ensures that that particular mapper is used here and not some other ObjectMapper in the project?